### PR TITLE
add spylon kernel to run scala notebook 

### PR DESCRIPTION
**Pull request by nvliyuan**
add spylon kernel to run scala notebook and fix a breaking link in toree.md

Signed-off-by: liyuan <yuali@nvidia.com>

### CODE REVIEW

**Comment by nvliyuan** [comment]
fix issue #43

File: docs/get-started/xgboost-examples/notebook/spylon.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+Get Started with XGBoost4J-Spark with Spylon Kernel Jupyter Notebook
+===================================================================
+
+This is a getting started guide to XGBoost4J-Spark using a [Spylon Kernel](https://pypi.org/project/spylon-kernel/) Jupyter notebook. 
+At the end of this guide, the reader will be able to run a sample notebook that runs on NVIDIA GPUs.
+
+Before you begin, please ensure that you have setup 
+a [Spark Standalone Cluster](/docs/get-started/xgboost-examples/on-prem-cluster/standalone-scala.md).
+
+It is assumed that the `SPARK_MASTER` and `SPARK_HOME` environment variables are defined and point to the master spark URL,
```
Comment (viadea) [inline docs/get-started/xgboost-examples/notebook/spylon.md:10]:
```suggestion
It is assumed that the `SPARK_MASTER` and `SPARK_HOME` environment variables are defined and point to the Spark Master URL, 
```

File: docs/get-started/xgboost-examples/notebook/spylon.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+This is a getting started guide to XGBoost4J-Spark using a [Spylon Kernel](https://pypi.org/project/spylon-kernel/) Jupyter notebook. 
+At the end of this guide, the reader will be able to run a sample notebook that runs on NVIDIA GPUs.
+
+Before you begin, please ensure that you have setup 
+a [Spark Standalone Cluster](/docs/get-started/xgboost-examples/on-prem-cluster/standalone-scala.md).
+
+It is assumed that the `SPARK_MASTER` and `SPARK_HOME` environment variables are defined and point to the master spark URL, 
+and the home directory for Apache Spark respectively.
+
+1. Install jupyter notebook with spylon-kernel.
```
Comment (viadea) [inline docs/get-started/xgboost-examples/notebook/spylon.md:13]:
```suggestion
1. Install Jupyter Notebook with spylon-kernel.
```

File: docs/get-started/xgboost-examples/notebook/spylon.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+   ``` bash
+   # Install notebook and spylon-kernel (Scala kernel for Jupyter Notebook), https://pypi.org/project/spylon-kernel/
+   # You can use spylon-kernel as Scala kernel for Jupyter Notebook. Do this when you want to work with Spark in Scala with a bit of Python code mixed in.
+   RUN pip3 install jupyter notebook spylon-kernel
+   RUN python -m spylon_kernel install
+   # Latest version breaks nbconvert: https://github.com/ipython/ipykernel/issues/422
+   RUN pip3 install ipykernel==5.1.1
+   ```
+2. Start up Jupiter notebook. You can debug from webUI http://your_ip:your_port with your password.
```
Comment (viadea) [inline docs/get-started/xgboost-examples/notebook/spylon.md:22]:
```suggestion
2. Start up Jupyter Notebook. You can debug from webUI http://your_ip:your_port with your password.
```

File: docs/get-started/xgboost-examples/notebook/spylon.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    
+    ``` bash
+    export JUPYTER_CONFIG_FILE=~/.jupyter/jupyter_notebook_config.py
+    
+    # Let's make the default password as '123456'
+    rm -rf `dirname $JUPYTER_CONFIG_FILE` && mkdir -p `dirname $JUPYTER_CONFIG_FILE` && echo """
+    c.NotebookApp.ip='*'
+    c.NotebookApp.password = u'sha1:cb0789ab252c:9b3f194578bc38c2e752e7a207754e4840280881'
```
Comment (viadea) [inline docs/get-started/xgboost-examples/notebook/spylon.md:30]: Let's mask the password hash.

File: docs/get-started/xgboost-examples/notebook/spylon.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    #    "version": "0.4.1"
+    #   }
+    #  },
+    #  "nbformat": 4,
+    #  "nbformat_minor": 5
+    # }
+    ```
+    You can also run python notebook with Spylon Kernel
+    ``` bash
+    # restart Jupiter notebook
```
Comment (viadea) [inline docs/get-started/xgboost-examples/notebook/spylon.md:81]:
```suggestion
    # restart Jupyter Notebook
```

File: examples/Spark-ETL+XGBoost/README.md [hunk 6]
```diff
@@ -29,6 +29,7 @@ which is used in the guides below.
     - [Databricks](../../docs/get-started/xgboost-examples/csp/databricks/databricks.md)
 - Getting started for Jupyter Notebook applications
     - [Apache Toree Notebook for Scala](../../docs/get-started/xgboost-examples/notebook/toree.md)
+    - [Spylon kernel Notebook for Scala](../../docs/get-started/xgboost-examples/notebook/spylon.md)
```
Comment (viadea) [inline examples/Spark-ETL+XGBoost/README.md:32]:
```suggestion
    - [Jupyter Notebook with spylon kernel for Scala](../../docs/get-started/xgboost-examples/notebook/spylon.md)
```

**Review by viadea — DISMISSED** [review]

**Comment by nvliyuan** [comment]
@viadea @GaryShen2008 I will close this pr and will draft a new one because some of the updated works are already included in #83

**Pull request closed without merging**